content/response-to-reviewers.md

```diff
@@ -246,11 +266,28 @@ Please see discussion at <https://github.com/greenelab/iscb-diversity-manuscript
 > Comments on the presentation:
 > \* The abstract lists only one conclusion in the last sentence: white scientists are overrepresented and non-white scientists are underrepresented. This strikes me as cherry picking of findings. The paper, if I understood correctly, also found that female scientists are not underrepresented. Given the current climate, wouldn't it also be a major conclusion that the honorees are reasonably distributed gender-wise? Instead, the conclusion is "but the proportion has not reached parity". This suddenly changes the focus from ISCB practices relative to the field to ISCB practices relative to the entire society, which is a different background distribution and one where ISCB has little influence on.
 
-Discussion at <https://github.com/greenelab/iscb-diversity-manuscript/issues/66>
+We agree with the reviewer that this observation should be added to the abstract.
+However, we do not believe that "the honorees are reasonably distributed gender-wise".
+While the data shows similar gender distributions between the group of honorees and authors, we noted in the Discussion session that other biases might contribute to an underestimation of female scientists in the field.
+Nevertheless, the proportions are far from parity.
+We have revised the abstract and added:
+
+"Although we observe similar gender distributions between the group of honorees and authors, other biases might contribute to an underestimation of female scientists in the field, and the proportions have not reached parity."
```

Comment:
```suggestion
"Although we observe similar gender distributions between the group of honorees and authors, the proportion of female scientists in the field has not yet reached parity."
```

Reply:
just made this match my suggestion above, so if you change it there - change it here 😁